src/github.com/getlantern/flashlight/main/main.go

```diff
@@ -53,7 +54,12 @@ func init() {
 }
 
 func logPanic(msg string) {
-	cfg, err := config.Init(flashlight.PackageVersion, *configdir, *stickyConfig, flagsAsMap())
+	// Request the config via either chained servers or direct fronted servers.
+	// We have to do the full version here because we're actually trying to
+	// submit this panic to our logs.
+	cf := util.NewChainedAndFronted(client.Addr)
+	configFetcher := config.NewFetcher(settings.GetUserID, settings.GetToken, cf)
```

Comment:
It's passing the funcs not the values. We then only pass the headers if the values are populated. See https://github.com/getlantern/lantern/pull/3806/files#diff-09edfa285ec7a6c867e5d7471691800aR105